demos/utils/gst.ts

```diff
@@ -31,7 +31,7 @@ export function gstStreamWebcam(ip: string, port: number, displayOutput: boolean
 
 function checkGstPlugins(plugins: string[]) {
   plugins.forEach(plugin => {
```

Comment:
this function is incorrect, exception will not be thrown in checkGstPlugins
convert it to regular for loop, or `await Promise.all(plugins.map(plugin => ...))`